### PR TITLE
Add onlinelibrary.wiley.com to allowlist

### PR DESCRIPTION
**Pull request by brifemu**
Due to issue #2490, I had included 'https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n1' into the current allow list.

The current allow list could bring us problems in the future because we are not controlling an exact error code and if for example tomorrow this site is closed we would not get notices because the current allow list doesn't difference between 404 or 503 error.

I will create another PR in the next weeks about this, after finish the new FAQ and more pending stuff.

---
Internal Tracking ID: [EXPOSUREAPP-12411](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12411)

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@brifemu 

I am happy to hear from you again.

I found several link running this branch so it seems that it hadn't been tested yet:

==================== Broken links ====================

```
[RESPONSE 404] http://localhost:8000/de/blog/2022-03-16-cwa-version-2-19/ on '/de/faq/' 
[RESPONSE 404] https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/mindestkriterien-sars-cov-2-antigentests-01-12-2020.pdf?__blob=publicationFile&v=7 on '/de/faq/' 
[RESPONSE 404] http://localhost:8000/en/blog/2022-03-16-cwa-version-2-19/ on '/en/faq/' 
[RESPONSE 404] https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/mindestkriterien-sars-cov-2-antigentests-01-12-2020.pdf?__blob=publicationFile&v=7 on '/en/faq/' 
[RESPONSE 503] https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n1 on 'http://localhost:8000/de/blog/2021-08-05-statistictiles' 
[RESPONSE 503] https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n1 on 'http://localhost:8000/en/blog/2021-08-05-statistictiles' 
```

I will deal with the blog and pei links separately.

**Comment by MikeMcC399** [comment]
- See issue https://github.com/corona-warn-app/cwa-website/issues/2555.

**Comment by MikeMcC399** [comment]
- See https://github.com/corona-warn-app/cwa-website/issues/2557

File: cypress/integration/check_links.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ context("Check for broken links", () => {
                   '/en/science/'
                 ]
     const allowlist = [
-      'https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999'
+      'https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999',
+      'https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n'
```
Comment (MikeMcC399) [inline cypress/integration/check_links.js:33]:
```suggestion
      'https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n1'
```

The URL was incomplete. 

- See original issue https://github.com/corona-warn-app/cwa-website/issues/2490

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
The allowlist URL is incomplete. See in-line comment (it should be `n1` not just `n` at the end of the URL). Note the error message for the correct URL to add to the allowlist:

`[RESPONSE 503] https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n1 on 'http://localhost:8000/en/blog/2021-08-05-statistictiles'`

**Comment by MikeMcC399** [comment]
The Cypress test suite will run without errors if the allowlist URL in this PR is corrected and the PRs

- #2555 and
- #2557 

are merged.

File: cypress/integration/check_links.js [hunk 2]
```diff
@@ -65,7 +66,8 @@ context("Check for broken links on entries", () => {
   const subpages = ['/de/blog/','/en/blog/','/de/science/', '/en/science/']
   const pagesToAvoid = ['/de/blog/', '/en/blog/', '/de/science/', '/en/science/', '/de/blog/archiv', '/en/blog/archive']
   const allowlist = [
-    'https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999'
+    'https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999',
+    'https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n'
```
Comment (MikeMcC399) [inline cypress/integration/check_links.js:70]:
```suggestion
    'https://onlinelibrary.wiley.com/doi/abs/10.2307/3315826.n1'
```

**Comment by MikeMcC399** [comment]
@brifemu 
Thank you for correcting the allowlist URLs.
Could you please also rebase on `master` or merge `master` into this branch so that the other URL fixes are included?

**Review by MikeMcC399 — APPROVED** [review]
@dsarkar 
Could we get this PR merged in order to correct the issue?

**Review by GisoSchroederSAP — APPROVED** [review]
Seems like a straight-foreward change.
Okay, from my side

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for rising this issue and reviewing it!

**Pull request merged**